picamera2/picamera2.py

```diff
@@ -293,6 +296,30 @@ def open_camera(self) -> None:
         else:
             raise RuntimeError("Failed to initialize camera")
 
+    def read_formats(self) -> None:
+        """Read the available sensor formats
+        """
+        raw_config = self.camera.generate_configuration([libcamera.StreamRole.Raw])
```

Comment:
I expect we'll change the function name in line with what I said earlier, and perhaps the comment could point out that the camera will get reconfigured by calling it!!